src/OpenTelemetry.ClrProfiler.Managed/Configuration/Settings.cs

```diff
@@ -164,5 +170,32 @@ internal static Settings FromDefaultSources()
 
             return new Settings(configurationSource);
         }
+
+        private static OtlpExportProtocol GetExporterOtlpProtocol(IConfigurationSource source)
+        {
+            // the defualt in SDK is grpc. It is oposite that here.
+            var exporterOtlpProtocol = source.GetString(ConfigurationKeys.ExporterOtlpProtocol);
+
+            if (exporterOtlpProtocol == "grpc")
+            {
+                return OtlpExportProtocol.Grpc;
+            }
+
+            return OtlpExportProtocol.HttpProtobuf;
+        }
+
+        private static Uri GetExporterOtlpEndpoint(IConfigurationSource source, OtlpExportProtocol otlpExportProtocol)
```

Comment:
do not we need to change the default only if `exporterOtlpEndpoint` is null or empty and `otlpExportProtocol == OtlpExportProtocol.HttpProtobuf` ? otherwise the SDK should provide good value

Reply:
Technically, you are right, but I expect to remove the code just after next release of OTEL .NET SDK.
https://github.com/open-telemetry/opentelemetry-dotnet/pull/2868

I would like to keep it as it is.